teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    @specialization = specialization
+    super(id, age, name, parent_permission: true)
```

Comment:
- [ ] Kindly consider to change your code taking the following lines in consideration:
  - you need to write `age, name = 'Unknown, parent_permission = true'` in the initialize parameters.
  - The super keyword should be on line 5 instead of line 6.
  - You dont need to write id in the super parameters as person's id would be given the teacher.

Reply:
Thank you for the helpful review. surely will come up with solution